File: plugin_fly_find/src/main/java/com/fly/tour/find/MainFindActivity.java
package com.fly.tour.find;

import com.fly.tour.common.base.BaseActivity;
import com.fly.tour.find.fragment.MainFindFragment;

public class MainFindActivity extends BaseActivity {


    @Override
    public int onBindLayout() {
        return R.layout.activity_find_main;
    }

    @Override
    public void initView() {
        getSupportFragmentManager().beginTransaction().replace(R.id.frame_content, MainFindFragment.newInstance()).commit();
    }

    @Override
    public void initData() {

    }
}

File: plugin_fly_find/src/main/java/com/fly/tour/find/MainFindActivity.kt
package com.fly.tour.find

import com.fly.tour.common.base.BaseActivity
import com.fly.tour.find.fragment.MainFindFragment

class MainFindActivity : BaseActivity() {
    override fun onBindLayout(): Int {
        return R.layout.activity_find_main
    }

    override fun initData() {
        supportFragmentManager.beginTransaction().replace(R.id.frame_content, MainFindFragment.Companion.newInstance()).commit()
    }
}